awgbuild.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # Copied from tungsten
-# export REACT_APP_GDC_DISPLAY_SLIDES=true
-export REACT_APP_SLIDE_IMAGE_ENDPOINT="https://api.gdc.cancer.gov/tile/"
+export REACT_APP_GDC_DISPLAY_SLIDES=true
```

Comment:
you don't want to enable this flag by default until the feature is ready to release, as that'd make it visible by the public.